cold-wallet/seed/secure-seed.ts

```diff
@@ -1,15 +1,39 @@
 import Config from '../../shared/config';
-import { Logger } from '../../shared/logger';
+import { Shamir } from '../../shared/shamir';
+import { Util } from '../../shared/util';
 
 export class SecureSeed {
-  private static logger = new Logger('Secure Seed');
-
   static async splitAndStore(seed: string[]): Promise<void> {
-    this.logger.info('"safely" secured following seed :D');
-    this.logger.info('', seed);
+    // generate shamir shares
+    const secret = seed.join(' ');
+    const shares = Shamir.split(secret, Config.wallet.seed.shareCount, Config.wallet.seed.thresholdCount);
+
+    // write to file
+    shares.forEach((share, i) => Util.writeFileRaw(Config.wallet.seed.fileName(i + 1), share));
```

Comment:
Why `i + 1`?

Reply:
Shamir uses 1-based indexing and I thought it might be a good idea to take this on 😉

Reply:
But in the end it just creates confusion ... I'll change it to 0-based.